src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  --one-color: #687c94;
+  --two-color: #3d4e61;
+  --light-color: #e1e7ed;
+  --white--color: #fff;
+  --focus-color: #f6f6f7;
+  --input-color: rgba(61, 78, 97, 0.1);
+}
+
+.input {
+  display: flex;
+  width: 100%;
+  line-height: 70px;
+  border-radius: 4px;
+  border: 1px solid var(--light-color);
+  background: var(--white--color);
+  box-shadow: 0 1px 8px var(--input-color);
+  margin: 20px 0;
+  background-image: url(../src/images/Search.svg);
+  background-size: 19px;
+  background-repeat: no-repeat;
+  position: relative;
+  background-position: 26px;
+  padding: 0 62px;
+  font-size: 16px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(70deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 900;
+}
+
+.input--small {
+  padding: 0 33px;
+  background-position: 12px;
+  height: 40px;
+  background-size: 11px;
+  font-size: 14px;
+}
+
+.label--small {
+  padding: 12px 33px 0;
+
+}
+
+.input--small:hover {
+  font-size: 14px;
+}
+
+::placeholder {
+  /* color: var(--two-color); */
```

Comment:
delete comments, also you need to write base class before applying pseudo class
`input::placeholder`